R/crew_launcher_lsf.R

```diff
@@ -0,0 +1,287 @@
+#' @title `r lifecycle::badge('experimental')` Create a launcher with
+#'   LSF workers.
+#' @export
+#' @family launchers
+#' @description Create an `R6` object to launch and maintain
+#'   workers as LSF jobs.
+#' @details WARNING: the `crew.cluster` LSF plugin is experimental
+#'   and has not actually been tested on a LSF cluster. Please proceed
```

Comment:
It seems right to keep this experimental, but thanks to you, I don't think we have to claim the LSF plugin has not been tested.